samples/texture_3d.html

```diff
@@ -131,12 +131,46 @@
 
         // -- Initialize texture
 
+        // var SIZE = 32;
+        // var data = new Float32Array(SIZE * SIZE * SIZE);
+        // for (var k = 0; k < SIZE; ++k) {
+        //     for (var j = 0; j < SIZE; ++j) {
+        //         for (var i = 0; i < SIZE; ++i) {
+        //             data[i + j * SIZE + k * SIZE * SIZE] = snoise([i, j, k]);
+        //         }
+        //     }
+        // }
+
+        // var texture = gl.createTexture();
+        // gl.activeTexture(gl.TEXTURE0);
+        // gl.bindTexture(gl.TEXTURE_3D, texture);
+        // gl.texParameteri(gl.TEXTURE_3D, gl.TEXTURE_BASE_LEVEL, 0);
+        // gl.texParameteri(gl.TEXTURE_3D, gl.TEXTURE_MAX_LEVEL, Math.log2(SIZE));
+        // gl.texParameteri(gl.TEXTURE_3D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
+        // gl.texParameteri(gl.TEXTURE_3D, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
+
+        // gl.texImage3D(
+        //     gl.TEXTURE_3D,  // target
+        //     0,              // level
+        //     gl.R32F,        // internalformat
+        //     SIZE,           // width
+        //     SIZE,           // height
+        //     SIZE,           // depth
+        //     0,              // border
+        //     gl.RED,         // format
+        //     gl.FLOAT,       // type
+        //     data            // pixel
+        //     );
+        
+        // gl.generateMipmap(gl.TEXTURE_3D);
+            
+            
         var SIZE = 32;
-        var data = new Float32Array(SIZE * SIZE * SIZE);
+        var data = new Uint8Array(3 * SIZE * SIZE * SIZE);
```

Comment:
You could continue to use a Uint8Array of just SIZE * SIZE * SIZE here. See below.